DEPLOYMENT.md

```diff
@@ -0,0 +1,128 @@
+# Dokku server setup
+
+## Create server
+
+* make a server (on Linode)
+  * needs at least 2GB RAM for the build step or it crashes out (with code 137 - OOM)
+
+## Create users
+
+Create all users as appropriate, e.g.
+
+```bash
+# create users
+adduser madwort
+# add sudo
+usermod -aG sudo madwort
+# add ssh key
+sudo su - madwort
+ssh-import-id gh:madwort
+```
+
+## Install Dokku
+
+based on https://dokku.com/docs/getting-started/installation/
+
+```bash
+# for debian systems, installs Dokku via apt-get
+wget -NP . https://dokku.com/install/v0.35.16/bootstrap.sh
+sudo DOKKU_TAG=v0.35.16 bash bootstrap.sh
+
+# add a key
+cat id_rsa.pub | dokku ssh-keys:add admin
+
+# install dokku-memcached plugin
+sudo dokku plugin:install https://github.com/dokku/dokku-memcached.git --name memcached
+dokku memcached:create microweb-memcached
+```
+
+## Create microweb app
+
+Create the app on the server:
+
+```bash
+dokku apps:create microweb
+```
+
+Push to the app from your local git repository:
+
+```bash
+git remote add dokku dokku@lfgssdemo:microweb
+git push dokku main
+# nb. if you're pushing a wip branch!
+git push dokku madwort/working-docker:main
+```
+
+Configure some things on the server:
+
+```bash
+dokku ports:add microweb http:80:5000
+dokku domains:add microweb lfgss.com
+dokku domains:add microweb www.lfgss.com
+dokku domains:add microweb *.microcosm.app
+
+dokku config:set --no-restart microweb DJANGO_SETTINGS_MODULE=microweb.settings
+dokku config:set --no-restart microweb PYTHONPATH=.
+dokku config:set --no-restart microweb MEMCACHE_HOST=dokku-memcached-microweb-memcached
+dokku config:set --no-restart microweb CLIENT_SECRET=123456
+dokku config:set --no-restart microweb SECRET_KEY=aoeui12345
```

Comment:
use single quotes if there are special chars in it (that bash will try to do something with)
```suggestion
dokku config:set --no-restart microweb SECRET_KEY='!aoeui12345@'
```